src/specific/s_filesystem.c

```diff
@@ -17,3 +26,14 @@ const char *S_File_GetGameDirectory()
     }
     return m_GameDir;
 }
+
+void S_File_CreateDirectory(const char *path)
+{
+    assert(path);
+    int32_t err_check = 0;
+#if defined(_WIN32)
+    err_check = _mkdir(path);
+#else
+    err_check = mkdir(path, 0664);
```

Comment:
Since we don't do anything with this variable, assignments are no longer needed.